docs/source/hardware/low_priority_faqs.md

```diff
@@ -0,0 +1,210 @@
+Submitting jobs to this partition will make your jobs preemptable. That means they may be stopped/suspended and re-queued at any time. We recommend that you use strategies such as checkpointing (see {ref}`checkpoint-jobs`) to take full advantage of the `lowpriority` partition. RC is working on detailed documentation (see {ref}`low-priority`) and training sessions that will be available later in the Fall 2023 semester, once the service is ready to be released.
+::::
+
+::::{dropdown} 3. How does re-queuing of preempted (terminated) low priority jobs work?
+Preempted jobs are put back in the default partition queue (`short`) and scheduled normally.
+::::
+
+::::{dropdown} 4. My servers are being used by others, how fast can we retrieve them?
+As a private partition owner you will always have the highest priority when accessing your own resources. When submitting jobs to your private partition, any low priority jobs currently running on your servers will be killed and sent back to the queue. You will retrieve your server(s) within approximately 30s.
+::::
+
+::::{dropdown} 5. I already have access to one or more of the large, long, and/or multigpu partition(s), do I benefit from using the `lowpriority` partition?
+
+The goal of `lowpriority` partition is to double the resources available to Discovery users. Hence, specifying the `lowpriority` partition in your SLURM job header gives your job a higher chance of being allocated resources, even when your first choice of partition is unavailable. For e.g., when you specify `#SBATCH --partition=short,lowpriority`, your job can start running on the `lowpriority` partition even when `short` is unavailable. 
+::::
+
+:::{button-link} faq.html
+:color: primary
+:align: center
+Back to the top
+:::
+
+## RC Policy
+
+::::{dropdown} 1. Can I opt-out?
+We would prefer if you did not, but can if it's really required due to constraints like funding stipulations. Opening up idle private resources that consume power and cooling makes the cluster more useful for everyone (including you). This is a common practice in HPC implemented by many other universities and national labs. Also, all private hardware purchases that were placed since 2019 came with the explicit understanding that they would be added to the `lowpriority` partition when it became available (see next question).
+::::
+
+::::{dropdown} 2. What is the understanding between RC and private partition owners?
+The following understanding applies to all individual private partition owners who purchased servers hosted at MGHPCC since 2019:
+
+RC racks, installs and maintains privately owned servers; RC purchases the equipment needed to connect these servers to the infiniband network fabric; RC pays for power and cooling to operate the servers during their entire lifetime; In return, private partition owners make their servers available to the `lowpriority` partition.
+::::
+
+::::{dropdown} 3. Are there policies or other formal documents available?
```

Comment:
We should remove this question until we have the MoU completed and distributed.

Reply:
I just removed that question and updated the numbering in the section.